lit_llama/model.py

```diff
@@ -13,12 +13,12 @@
 
 
 def build_rope_cache(seq_len: int, n_elem: int, dtype: torch.dtype, base: int = 10000) -> torch.Tensor:
-    """Enhanced Transformer with Rotary Position Embedding.
+    """This is the `build_rope_cache` implementation we initially intended to use, but it is numerically not
+    exactly equivalent to the one in the Meta model. We keep it here for posterity.
```

Comment:
Going back to this implementation means we won't be able to load other weights directly and use the model. One could 1) add a flag that switches between the two implementation 2) have this one but then somehow adapt/correct them with tuning. Any other ideas?

Reply:
My bad, I messed up with my earlier edits. I wasn't trying to go back to the old rope implementation.